zapcore/facility.go

```diff
@@ -41,6 +41,19 @@ type Facility interface {
 	Write(Entry, []Field) error
 }
 
+type nopFacility struct{}
+
+// NopFacility returns a no-op Facility.
+func NopFacility() Facility { return nopFacility{} }
+
+func (nopFacility) Enabled(Level) bool { return false }
+
```

Comment:
optional: if you remove the blank lines between the methods, go will align the bodies. since the implementations are not complex, think it might be worth removing the extra whitespace